src/utils/getValidSubjectCategoryIconById.ts

```diff
@@ -17,8 +17,21 @@ const subjectCategoryIconMap: Record<number, string> = {
   17: "theology",
 };
 
-export function getValidSubjectCategoryIconById(id: number): OakIconName {
-  const subjectSlug = subjectCategoryIconMap[id];
+const allIconMap: Record<string, OakIconName> = {
+  "religious-education": "subject-religious-education",
+  english: "books",
+  science: "rocket",
```

Comment:
@orangemug These should be the subject icons (e.g. `subject-science`) - this is the wrong rocket:
<img width="1440" alt="Screenshot 2025-03-31 at 16 31 53" src="https://github.com/user-attachments/assets/677058ba-1e3e-490d-8c5d-062bbd4d04d5" />

Reply:
@orangemug Need to do the same for English, right?